### PR TITLE
Fixing postman endpoint to use api instead of app

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

Fixes endpoint for some postman endpoint to use api.datadoghq.com instead of app.datadoghq.com

### CODE REVIEW

File: static/resources/json/datadog_collection.json [hunk 1]
```diff
@@ -360,13 +360,13 @@
 								],
 								"body": {
 									"mode": "raw",
-									"raw": "{\n    \"message\": \"<MESSAGE>\",\n    \"handle\": \"<DATADOG_HANDLE>\"\n}"
+									"raw": "{\n    \"message\": \"<MESSAGE>\",\n    \"handle\": \"pierre.guceski@datadoghq.com\"\n}"
```
Comment (ruthnaebeck) [inline static/resources/json/datadog_collection.json]: Should this be here?

File: static/resources/json/datadog_collection.json [hunk 2]
```diff
@@ -439,10 +455,10 @@
 								}
 							],
 							"cookie": [],
-							"body": "{\n    \"errors\": [\n        \"The value provided for parameter 'handle' is invalid\"\n    ]\n}"
+							"body": "{\n    \"comment\": {\n        \"url\": \"/event/event?id=4889126983613267970\",\n        \"resource\": \"/api/v1/comments/4889126983613267970\",\n        \"message\": \"<MESSAGE>\",\n        \"handle\": \"test@datadoghq.com\",\n        \"id\": 4889126983613267970\n    }\n}"
```
Comment (ruthnaebeck) [inline static/resources/json/datadog_collection.json]: Are you sure?

File: static/resources/json/datadog_collection.json [hunk 3]
```diff
@@ -618,7 +618,7 @@
 									"raw": "{\n    \"message\": \"<NEW_COMMENT_MESSAGE>\",\n    \"handle\":\"pierre.guceski@datadoghq.com\"\n}"
 								},
```
Comment (ruthnaebeck) [inline static/resources/json/datadog_collection.json]: Your email address

File: static/resources/json/datadog_collection.json [hunk 4]
```diff
@@ -618,7 +618,7 @@
...
 								"url": {
-									"raw": "https://api.datadoghq.com/api/v1/comments/:COMMENT_ID?api_key={{dd_api_key}}&application_key={{dd_app_key}}",
+									"raw": "https://app.datadoghq.com/api/v1/comments/:COMMENT_ID?api_key={{dd_api_key}}&application_key={{dd_app_key}}",
```
Comment (ruthnaebeck) [inline static/resources/json/datadog_collection.json]: api vs app?

File: static/resources/json/datadog_collection.json [hunk 5]
```diff
@@ -735,7 +735,7 @@
 									"raw": "{\n    \"message\": \"My new comment message.\"\n}"
 								},
 								"url": {
-									"raw": "https://api.datadoghq.com/api/v1/comments/:EVENT_ID?api_key={{dd_api_key}}&application_key={{dd_app_key}}",
+									"raw": "https://app.datadoghq.com/api/v1/comments/:EVENT_ID?api_key={{dd_api_key}}&application_key={{dd_app_key}}",
```
Comment (ruthnaebeck) [inline static/resources/json/datadog_collection.json]: api vs app?

File: static/resources/json/datadog_collection.json [hunk 6]
```diff
@@ -1256,9 +1256,9 @@
 									],
 									"variable": [
 										{
-											"description": "ID of the Dashboard to update",
 											"key": "DASHBOARD_ID",
-											"value": "civ-zbj-ag5"
+											"value": "civ-zbj-ag5",
```
Comment (ruthnaebeck) [inline static/resources/json/datadog_collection.json]: should this be `{{dashboard_id}}` ?

File: static/resources/json/datadog_collection.json [hunk 7]
```diff
@@ -1426,9 +1426,9 @@
 									],
 									"variable": [
 										{
-											"description": "Id of the Dashboar to delete",
 											"key": "DASHBOARD_ID",
-											"value": "civ-zbj-ag5"
+											"value": "civ-zbj-ag5",
```
Comment (ruthnaebeck) [inline static/resources/json/datadog_collection.json]: should this be `{{dashboard_id}}` ?

File: static/resources/json/datadog_collection.json (hunk 1 above)
Comment (l0k0ms) [reply static/resources/json/datadog_collection.json]: that's weird....

File: static/resources/json/datadog_collection.json [hunk 8]
```diff
@@ -1596,9 +1596,9 @@
 									],
 									"variable": [
 										{
-											"description": "Id of the Dashboard to GET",
 											"key": "DASHBOARD_ID",
-											"value": "civ-zbj-ag5"
+											"value": "civ-zbj-ag5",
```
Comment (ruthnaebeck) [inline static/resources/json/datadog_collection.json]: `{{dashboard_id}}`?

File: static/resources/json/datadog_collection.json [hunk 9]
```diff
@@ -2289,6 +2289,64 @@
...
+										{
+											"key": "application_key",
+											"value": "{{dd_app_key}}",
+											"description": "Your Datadog application key"
+										}
+									],
+									"variable": [
+										{
+											"key": "DASHBOARD_LIST_ID",
+											"value": "4741",
```
Comment (ruthnaebeck) [inline static/resources/json/datadog_collection.json]: `{{dashboard_id}}`?

File: static/resources/json/datadog_collection.json [hunk 10]
```diff
@@ -3184,7 +3184,7 @@
 									"raw": "{\n      \"scope\": \"host:i-123\"\n}"
 								},
 								"url": {
-									"raw": "https://api.datadoghq.com/api/v1/downtime/cancel/by_scope?api_key={{dd_api_key}}&application_key={{dd_app_key}}",
+									"raw": "https://app.datadoghq.com/api/v1/downtime/cancel/by_scope?api_key={{dd_api_key}}&application_key={{dd_app_key}}",
```
Comment (ruthnaebeck) [inline static/resources/json/datadog_collection.json]: `api`?

File: static/resources/json/datadog_collection.json [hunk 11]
```diff
@@ -4091,7 +4091,7 @@
 									"raw": "{\n\t\"title\":\"Something big happened!\",\n\"text\":\"And let me tell you all about it here!\",\n\"tags\":\"environment:prod\"\n}"
 								},
 								"url": {
-									"raw": "https://api.datadoghq.com/api/v1/events/:EVENT_ID?api_key={{dd_api_key}}&application_key={{dd_app_key}}",
+									"raw": "https://app.datadoghq.com/api/v1/events/:EVENT_ID?api_key={{dd_api_key}}&application_key={{dd_app_key}}",
```
Comment (ruthnaebeck) [inline static/resources/json/datadog_collection.json]: `api`?

File: static/resources/json/datadog_collection.json [hunk 12]
```diff
@@ -5575,7 +5575,7 @@
 									"raw": ""
 								},
 								"url": {
-									"raw": "https://api.datadoghq.com/api/v1/integration/azure?api_key={{dd_api_key}}&application_key={{dd_app_key}}",
+									"raw": "https://app.datadoghq.com/api/v1/integration/azure?api_key={{dd_api_key}}&application_key={{dd_app_key}}",
```
Comment (ruthnaebeck) [inline static/resources/json/datadog_collection.json]: `api`?

File: static/resources/json/datadog_collection.json [hunk 13]
```diff
@@ -5620,7 +5620,7 @@
 									"raw": "{\n    \"tenant_name\": \"<AZURE_TENANT_NAME>\",\n    \"client_id\": \"<AZURE_CLIENT_ID>\",\n    \"client_secret\": \"<AZURE_CLIENT_SECRET>\",\n    \"host_filters\": \"<KEY_1>:<VALUE_1>,<KEY_2>:<VALUE_2>\"\n}"
 								},
 								"url": {
-									"raw": "https://api.datadoghq.com/api/v1/integration/azure?api_key={{dd_api_key}}&application_key={{dd_app_key}}",
+									"raw": "https://app.datadoghq.com/api/v1/integration/azure?api_key={{dd_api_key}}&application_key={{dd_app_key}}",
```
Comment (ruthnaebeck) [inline static/resources/json/datadog_collection.json]: `api`?

File: static/resources/json/datadog_collection.json [hunk 14]
```diff
@@ -5665,7 +5665,7 @@
 									"raw": "{\n    \"tenant_name\": \"<AZURE_TENANT_NAME>\",\n    \"client_id\": \"<AZURE_CLIENT_ID>\",\n    \"host_filters\": \"<KEY_1>:<VALUE_1>,<KEY_2>:<VALUE_2>\"\n}"
 								},
 								"url": {
-									"raw": "https://api.datadoghq.com/api/v1/integration/azure?api_key={{dd_api_key}}&application_key={{dd_app_key}}",
+									"raw": "https://app.datadoghq.com/api/v1/integration/azure?api_key={{dd_api_key}}&application_key={{dd_app_key}}",
```
Comment (ruthnaebeck) [inline static/resources/json/datadog_collection.json]: `api`?

File: static/resources/json/datadog_collection.json [hunk 15]
```diff
@@ -5710,7 +5710,7 @@
 									"raw": "{\n    \"tenant_name\": \"<AZURE_TENANT_NAME>\",\n    \"client_id\": \"<AZURE_CLIENT_ID>\"\n}"
 								},
 								"url": {
-									"raw": "https://api.datadoghq.com/api/v1/integration/azure?api_key={{dd_api_key}}&application_key={{dd_app_key}}",
+									"raw": "https://app.datadoghq.com/api/v1/integration/azure?api_key={{dd_api_key}}&application_key={{dd_app_key}}",
```
Comment (ruthnaebeck) [inline static/resources/json/datadog_collection.json]: `api`? 

Several more below this

File: static/resources/json/datadog_collection.json (hunk 6 above)
Comment (l0k0ms) [reply static/resources/json/datadog_collection.json]: it's an example, that's why

File: static/resources/json/datadog_collection.json (hunk 7 above)
Comment (l0k0ms) [reply static/resources/json/datadog_collection.json]: ditto

File: static/resources/json/datadog_collection.json (hunk 8 above)
Comment (l0k0ms) [reply static/resources/json/datadog_collection.json]: ditto

File: static/resources/json/datadog_collection.json (hunk 9 above)
Comment (l0k0ms) [reply static/resources/json/datadog_collection.json]: ditto

File: static/resources/json/datadog_collection.json (hunk 10 above)
Comment (l0k0ms) [reply static/resources/json/datadog_collection.json]: that's weird.. i know but when looking at the postman collection the values are correct

File: static/resources/json/datadog_collection.json (hunk 2 above)
Comment (l0k0ms) [reply static/resources/json/datadog_collection.json]: yup it's an example

File: static/resources/json/datadog_collection.json (hunk 3 above)
Comment (l0k0ms) [reply static/resources/json/datadog_collection.json]: fixed

**Pull request merged**